Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/Airport.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Ibistic.Public.OpenAirportData
+{
+    public sealed class Airport
+    {
+        public string Name { get; internal set; }
+        public string City { get; internal set; }
+        public string CountryName { get; internal set; }
+        public string CountryAlhpa2 { get; internal set; }
+        public string CountryAlpha3 { get; internal set; }
+        public string IataCode { get; internal set; }
+        public string IcaoCode { get; internal set; }
+        public double Latitude { get; internal set; }
+        public double Longitude { get; internal set; }
+        public float AltitudeInFeet { get; internal set; }
+        public float TimezoneUtcOffset { get; internal set; }
+        public DaylightSavingsTimeType DaylightSavingsTime { get; internal set; }
+        public string TimezoneName { get; internal set; }
```

Comment:
Timezone is typically written TimeZone in the CLR.

Reply:
Fixing.